services/app-web/src/components/Form/ErrorSummary/ErrorSummary.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames'
+import classnames from 'classnames'
+import React, { useEffect, useRef } from 'react'
+import styles from './ErrorSummary.module.scss'
+
+type ErrorSummaryMessageProps = {
+    errorKey: string,
+    message: string
+}
+const ErrorSummaryMessage = ({
+    errorKey, message
+}: ErrorSummaryMessageProps): React.ReactElement => {
+    return (
+        <button
+            type="button"
```

Comment:
I remember there being a long discussion about Button vs. Link in the A11y course we took. since this is navigating us to somewhere one the page would it make more sense as a link?

Reply:
I think the link guidance is that you have to have a value for `href` for it to be a link. We have `id` attributes on the inputs, so I can change these to links and use those anchors.